src/i18n/translations/nb.ts

```diff
@@ -164,9 +164,10 @@ export const messages = {
   contactFieldsFurtherDetailsTitle: 'Detaljer',
   contactFieldsPhoneTitle: 'Telefon',
   contactFieldsUrlTitle: 'URL',
+  urlOrPhoneMustBeFilled: 'Både telefon og URL kan ikke være tomme',
```

Comment:
Kanskje vri denne til å være positiv?
```suggestion
  urlOrPhoneMustBeFilled: 'URL eller telefonnummer må utfylles',
```

Reply:
Ja, enig. Men tenkte at brukeren da tror at man bare kan fylle ut den ene?